test/rules_test.js

```diff
@@ -180,27 +179,27 @@ describe('applyRules', () => {
     });
     it('multiply fractions', () => {
         const tests = [
-            ['2 / 3 * 2 / 3', '2 * 2 / 3 * 3'],
-            ['x / 2 * x / 2', 'x * x / 2 * 2'],
-            ['(x + 1) / 2 * (x + 1) / 2', '(x + 1) * (x + 1) / 2 * 2'],
-            ['x^((x + 1) / 2 * (x + 1) / 2)', 'x^(x + 1) * (x + 1) / 2 * 2'],
+            ['2 / 3 * 2 / 3', '(2 * 2) / (3 * 3)'],
+            ['x / 2 * x / 2', '(x * x) / (2 * 2)'],
+            ['(x + 1) / 2 * (x + 1) / 2', '((x + 1) * (x + 1)) / (2 * 2)'],
+            ['x^((x + 1) / 2 * (x + 1) / 2)', 'x^((x + 1) * (x + 1)) / (2 * 2)'],
         ];
         tests.forEach(t => assert.equal(applyRuleString(rules.MULTIPLY_FRACTIONS, t[0]), t[1]));
     });
-    it('simplfy division', () => {
+    it('simplify division', () => {
         const tests = [
-            ['2 / 3 / 4', '2 / 3 * 4'],
-            ['x / 2 / 2', 'x / 2 * 2'],
-            ['(x + 1) / 2 / (x + 1)', '((x + 1)) / 2 * (x + 1)'],
-            ['x^((x + 1) / 2 / 2)', 'x^((x + 1)) / 2 * 2'],
+            ['2 / 3 / 4', '2 / (3 * 4)'],
+            ['x / 2 / 2', 'x / (2 * 2)'],
+            ['(x + 1) / 2 / (x + 1)', '(x + 1) / (2 * (x + 1))'],
+            ['x^((x + 1) / 2 / 2)', 'x^(x + 1) / (2 * 2)'],
```

Comment:
I printed out the ast for the test case:

`{
    "type": "Apply",
    "op": "pow",
    "args": [
        {
            "type": "Identifier",
            "name": "x",
            "loc": {
                "start": 0,
                "end": 1
            }
        },
        {
            "type": "Apply",
            "op": "div",
            "args": [
                {
                    "type": "Apply",
                    "op": "add",
                    "args": [
                        {
                            "type": "Identifier",
                            "name": "x",
                            "loc": {
                                "start": 4,
                                "end": 5
                            }
                        },
                        {
                            "type": "Number",
                            "value": "1",
                            "loc": {
                                "start": 8,
                                "end": 9
                            }
                        }
                    ],
                    "loc": {
                        "start": 4,
                        "end": 9
                    }
                },
                {
                    "type": "Apply",
                    "op": "mul",
                    "args": [
                        {
                            "type": "Number",
                            "value": "2",
                            "loc": {
                                "start": 13,
                                "end": 14
                            }
                        },
                        {
                            "type": "Number",
                            "value": "2",
                            "loc": {
                                "start": 17,
                                "end": 18
                            }
                        }
                    ],
                    "loc": {
                        "start": 6,
                        "end": 13
                    }
                }
            ],
            "loc": {
                "start": 0,
                "end": 13
            }
        }
    ],
    "loc": {
        "start": 0,
        "end": 18
    }
}`